docs/source/guide/index.md

```diff
@@ -81,8 +81,30 @@ The component parts of Label Studio are available as modular extensible packages
 | [Data Manager](https://github.com/heartexlabs/dm2) | JavaScript web app using [React](https://reactjs.org/) | Manage data and tasks for labeling. |
 | [Machine Learning Backends](https://github.com/heartexlabs/label-studio-ml-backend) | Python | Predict data labels at various parts of the labeling process. |
 
+The following figure shows the different components in Label Studio architecture.
 <br>
-<div style="margin:auto; text-align:center;"><img src="/images/ls-modules-scheme.png" style="opacity: 0.8"/></div>
+<div style="margin:auto; text-align:center;"><img src="/images/ls-modules-scheme.png" style="opacity: 0.8" class="gif-border"/></div>
+<i>Figure 1: Overview of Label Studio architecture.</i>
+
+
+The following figure shows the labeled UI elements in Workspace view.
```

Comment:
better to write "project list", not "workspace view"

Reply:
"labeled UI elements" sounds weird for me, because in project list and in data manager there is no "labeled elements".

I would write shortly everywhere:
Project list view, Quick view, Data manager view

Reply:
As we scale up, more views will be implemented and docs maintenance is a daily task. I would just call out the view like `Workspace view`.